### PR TITLE
[ALLUXIO-1600] update docs for the exceptions

### PR DESCRIPTION
**Pull request by shimingfei**
https://tachyon.atlassian.net/browse/ALLUXIO-1600

### CODE REVIEW

File: core/server/src/main/java/alluxio/web/WebInterfaceDownloadServlet.java [hunk 1]
```diff
@@ -104,6 +104,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
    * @param response the {@link HttpServletResponse} object
    * @throws FileDoesNotExistException if the file does not exist
    * @throws IOException if an I/O error occurs
+   * @throws AlluxioException if an unexpected Alluxio exception is thrown
```
Comment (aaudiber) [inline core/server/src/main/java/alluxio/web/WebInterfaceDownloadServlet.java]: this also throws InvalidPathException

**Comment by aaudiber** [comment]
Thanks for the PR @shimingfei, I've left a comment.

**Comment by yupeng9** [comment]
@shimingfei any update on this PR?

**Comment by shimingfei** [comment]
@yupeng9 
Sorry for late reply, I have updated the PR, Thanks!

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**